src/gene/etl/ensembl.py

```diff
@@ -265,7 +180,7 @@ def _add_meta(self) -> None:
             "/legal/disclaimer.html",
             version=self._version,
             data_url={
-                "genome_annotations": f"ftp://{self._host}/{self._data_dir}Homo_sapiens.{self._assembly}.{self._version}.gff3.gz"
+                "genome_annotations": f"ftp://ftp.ensembl.org/pub/current_gff3/homo_sapiens/Homo_sapiens.{self._assembly}.{self._version}.gff3.gz"
```

Comment:
I think for the data url we should use `ftp://ftp.ensembl.org/pub/release-{self._version}/gff3/homo_sapiens/Homo_sapiens.{self._assembly}.{self._version}.gff3.gz`. Since the current_gff3 will constantly be updated.

Reply:
This is just a suggestion, the only thing that actually needs changed is removing the bool from docstring

Reply:
yeah that is a good point